packages/desktop/views/dashboard/drawers/dapp-config/views/ConfirmConnectionDrawer.svelte

```diff
@@ -75,15 +75,12 @@
                     $sessionProposal.params.requiredNamespaces,
                     $sessionProposal.params.optionalNamespaces
                 )
-            await clearOldPairings($sessionProposal.params.proposer.metadata.url)
+            await clearOldPairings(dappUrl)
             await approveSession($sessionProposal, supportedNamespaces)
-            persistDappNamespacesForDapp($sessionProposal.params.proposer.metadata.url, supportedNamespaces)
+            persistDappNamespacesForDapp(dappUrl, supportedNamespaces)
             $sessionProposal = undefined
 
-            showNotification({
-                variant: 'success',
-                text: localize('notifications.newDappConnection.success'),
-            })
+            openPopup({ id: PopupId.SuccessfulDappConnection, props: { url: dappUrl } })
```

Comment:
do we want this also on each dapp request, ie sign message/tx?

Reply:
Question for Andrew tomorrow. I would leave it for now, since this pop-up guides them back to the dApp for interaction with the dApp itself.